cypress/integration/login.js

```diff
@@ -8,20 +8,95 @@ describe('Sourcebooks login', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    it('Should be able to login with role Team Lead', function () {
 
         cy.visit('/');
         cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
+        cy.get('[aria-label="Ieva Stonkaitė"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="User"]').click();
+        cy.get('[aria-label="Team Lead"]').click();
         cy.get('[type="submit"]').click();
 
         cy.url().should('include', '/time-logging');
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
-        cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.user-info__title').contains('Ieva Stonkaite');
+        cy.get('.main-nav').find('li').should('have.length', 2);
+    })
+
+    it('Should show calendar date', function () {
+        cy.get('.calendar--selected').find('span').contains(new Date().getDate());
+    })
+
+    it('Should verify all user roles can log in and should see appropriate tabs - USER', function () {
+    
+        cy.visit('/');
+        cy.get('[id="loginForm.userId"]').click({force:true});
+        cy.get('[aria-label="Ieva Stonkaitė"]').click();
+        cy.get('[id="loginForm.role"]').click({force:true});
+        cy.get('[aria-label="User"]').click();
+        cy.get('[type="submit"]').click();
+        cy.get('.user-info__title').contains('Ieva Stonkaite');
+        cy.get('.main-nav').find('li').should('have.length', 1); //tabu skaicius

```

Comment:
If you add comments they should be in english (as test names and everything else is in english). In this case i don't think this comment is needed as the person who will read code will know what this does and what "1" means :)